models/player.go

```diff
@@ -1,6 +1,146 @@
 package models
 
+import (
+	"fmt"
+	"github.com/TeamPlayTF/Server/database"
+	"gopkg.in/mgo.v2/bson"
+	"time"
+)
+
 type Player struct {
-	steamid string //Players steam ID
-	name    string //Player name
+	Id      bson.ObjectId `bson:"_id,omitempty"        json:"id"`      // MongoDB ID
```

Comment:
Only noticed this after merging but since you created getters and setters for Player, there's no point in making its fields public - they should probably be lowercase